_test/test_mpi/test_parallel_call.m

```diff
@@ -0,0 +1,78 @@
+classdef test_parallel_call < TestCase
+
+    properties
+        sqw_obj;
+        stored_hpc;
+        stored_par;
+    end
+
+    methods
+
+        function obj = test_parallel_call(name)
+            if ~exist('name', 'var')
+                name = 'test_parallel_call';
+            end
+            obj = obj@TestCase(name);
+
+            pths = horace_paths();
+            data_dir = pths.test_common;
+            test_sqw_file = fullfile(data_dir,'sqw_2d_2.sqw');
+            obj.sqw_obj = sqw(test_sqw_file);
+
+            hc = hpc_config();
+            obj.stored_hpc = hc.get_data_to_store();
+            pc = parallel_config();
+            obj.stored_par = pc.get_data_to_store();
+
+
+        end
+
+        function tearDown(obj)
+            set(hpc_config, obj.stored_hpc);
+            set(parallel_config, obj.stored_par);
+        end
+
+        function test_parallel_call_herbert_sqw_eval(obj)
+            pc = parallel_config();
+            pc.parallel_cluster = 'herbert';
+            pc.parallel_workers_number = 2;
```

Comment:
at least for compiled mpi, 2 workers is sometimes considered a special case which does not cover all eventualities. Is it worth doing n=4 tests?